src/session/session_api.c

```diff
@@ -653,8 +653,9 @@ __session_join(WT_SESSION *wt_session, WT_CURSOR *join_cursor,
 	WT_INDEX *idx;
 	WT_SESSION_IMPL *session;
 	WT_TABLE *table;
-	uint64_t bloom_bit_count, bloom_hash_count, count;
-	uint32_t flags, range;
+	uint8_t flags;
+	uint32_t bloom_bit_count, bloom_hash_count, range;
+	uint64_t count;
```

Comment:
@ddanderson swap `flags` and `count` lines so that they're in decreasing size order.